### ISSUE TITLE
Investigate default runtime-images based on non docker.io images

### ISSUE THREAD

**Issue by lresende** [elyra-ai/elyra#1234]
**Describe the issue**
With the docker.io rate limit restrictions, we should look for images on quay.io or other more permissible repositories and update the sample runtime images we add to the elyra deployment

**Comment by ConverJens** [comment]
@lresende Would it be possible to also add the option to specify a private repository? On-prem installations are usually closed off from internet access and restricting to only public repositories means Elyra can't be used. I could probably work on a PR for this.

**Comment by akchinSTC** [comment]
the pre-approved/default runtime images that elyra provides have been duplicated and push ed to the quay.io registry as a workaround here : https://quay.io/organization/elyra

@ConverJens - that would be fantastic. Please let us know if you have any questions! We have a weekly dev meeting tomorrow 2/4 at 9am PST if you want to chat there as well.

**Comment by ConverJens** [comment]
@akchinSTC So it turned out that the feature I was looking for was already implemented but the documentation was a bit misleading. This was fixed in another PR: https://github.com/elyra-ai/elyra/pull/1258. So I won't be opening a PR for this. Sotty!

**Comment by akchinSTC** [comment]
@ConverJens, great! no worries about the PR, as long as you are unblocked. Please let us know/open another issue if any other documentation is coming across as ambiguous. 

@lresende, should update our default runtime images to pull from quay.io and close this issue?

**Comment by akchinSTC** [comment]
All default runtime images have been pushed to https://quay.io/organization/elyra as well as docker hub. 
Elyra has also been added as exempt from the docker hub rate limitations. 
Closing this issue.
